### PR TITLE
Add globals to TableParallelize

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by tpoterba** [comment]
This is necessary for part of the MatrixIR lowering project: comment in #4905

**Comment by cseed** [comment]
@jbloom22 bump.  This has been waiting for two weeks.

**Comment by jbloom22** [comment]
I apologize for losing track, I will review it today.

File: hail/python/hail/ir/base_ir.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def __init__(self, *children):
     def aggregations(self):
         if self._aggregations is None:
             self._aggregations = [agg for child in self.children for agg in child.aggregations]
-        return self._aggregations
+        return self._aggregationsta
```
Comment (jbloom22) [inline hail/python/hail/ir/base_ir.py]: typo?

File: hail/python/hail/ir/table_ir.py [hunk 2]
```diff
@@ -173,15 +173,15 @@ def render(self, r):
 
 
 class TableParallelize(TableIR):
-    def __init__(self, rows, n_partitions):
+    def __init__(self, rows_and_globals, n_partitions):
```
Comment (jbloom22) [inline hail/python/hail/ir/table_ir.py]: make the change in Python too: `rows_and_global`

File: hail/src/main/scala/is/hail/expr/ir/Parser.scala [hunk 3]
```diff
@@ -838,8 +838,8 @@ object IRParser {
         TableMultiWayZipJoin(children, dataName, globalsName)
       case "TableParallelize" =>
         val nPartitions = opt(it, int32_literal)
-        val rows = ir_value_expr(env)(it)
-        TableParallelize(rows, nPartitions)
+        val rowsAndGlobals = ir_value_expr(env)(it)
```
Comment (jbloom22) [inline hail/src/main/scala/is/hail/expr/ir/Parser.scala]: and you missed some

File: hail/python/hail/ir/table_ir.py [hunk 4]
```diff
@@ -173,15 +173,15 @@ def render(self, r):
 
 
 class TableParallelize(TableIR):
-    def __init__(self, rows, n_partitions):
+    def __init__(self, rows_and_global, n_partitions):
         super().__init__()
-        self.rows = rows
+        self.rows_and_globals = rows_and_global
```
Comment (jbloom22) [inline hail/python/hail/ir/table_ir.py]: missed here

File: hail/python/hail/ir/table_ir.py [hunk 5]
```diff
@@ -173,15 +173,15 @@ def render(self, r):
...
         self.n_partitions = n_partitions
 
     def render(self, r):
         return '(TableParallelize {} {})'.format(
             self.n_partitions,
-            r(self.rows))
+            r(self.rows_and_globals))
```
Comment (jbloom22) [inline hail/python/hail/ir/table_ir.py]: and here

**Pull request merged**